README.md

```
@@ -87,6 +60,52 @@
+</details>
+
+<details>	
+  <br />
+  <summary><b>⚙️ Things I use to get stuff done</b></summary>
+  	<ul>
+  	  <li><b>OS:</b> MacOS / Linux</li>
+  	  <li><b>Browser: </b> Firefox / Chrome Browser</li>
+	  <li><b>Code Editor:</b> Visual Studio Code / Neovim </li>
+	  <li><b>To Stay Updated:</b> Dev.to, Medium, Twitter and Tech YouTube Channels</li>
+	</ul>
+	<b>Read more about it <a target="_blank" rel="norefferer noopener" href="https://akashrajpurohit.com/uses/?ref=github-profile-readme">here</a></b>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)